doc/man1/enc.pod

```diff
@@ -54,47 +54,47 @@ List all supported ciphers.
 
 =item B<-in filename>
 
-the input filename, standard input by default.
+The input filename, standard input by default.
```

Comment:
What really stood out was when the description was multiple sentences like:

> the password source. For more information about the...

which is inconsistent.